src/plugins/intel_cpu/src/nodes/transpose.cpp

```diff
@@ -111,7 +111,7 @@ void Transpose::initSupportedPrimitiveDescriptors() {
             supportedPrimitiveDescriptorsBuilder(config, transposeParams);
         }
 #endif // OPENVINO_ARCH_X86_64
-        if (prec == ov::element::f32 || prec == ov::element::f16 || prec == ov::element::i8 || prec == ov::element::u8) {
+        if (prec == ov::element::f32 || prec == ov::element::f16 || prec == ov::element::i8 || prec == ov::element::u8 || prec == ov::element::bf16) {
```

Comment:
Why add `bf16` condition?

Reply:
The master had this "Precision::BF16"
https://github.com/openvinotoolkit/openvino/blob/369831caef55efe1f2d87889d3997f93c53ab6b8/src/plugins/intel_cpu/src/nodes/transpose.cpp#L114